installing/installing_aws/manually-creating-iam.adoc

```diff
@@ -0,0 +1,134 @@
+[id="manually-creating-iam"]
+= Manually creating IAM
+include::modules/common-attributes.adoc[]
+:context: manually-creating-iam
+
+toc::[]
+
+[id="manually-create-iam_{context}"]
+== Manually create IAM
+
+The Cloud Credential Operator can be put into manual mode prior to installation
+in environments where the cloud identity and access management (IAM) APIs are
+not reachable, or the administrator prefers not to store an admin level
+credential secret in the cluster `kube-system` namespace.
```

Comment:
Can you think of a good spot to try to steer people away from this and towards using the admin cred normally? We consider this by far the best practice. Maybe some warnings here?

Reply:
There's also the option of removing the admin cred post-install. This allows the auto-managed mine-grained creds, but also doesn't leave the admin creds in the cluster after they aren't needed. (This does complicate upgrades if updated credentials are needed and CCO cannot update them w/o the admin creds).
cc @dgoodwin